### ISSUE TITLE
[BUG] ASLTotalAcquiredPairsASLContextLength check is incorrect

### ISSUE THREAD

**Issue by tsalo** [bids-standard/bids-specification#1975]
### Describe your problem in detail.

TotalAcquiredPairs is compared against the number of rows in aslcontext, but the TotalAcquiredPairs value should be the number of control-label pairs, while the aslcontext file should have one row for each volume.

### Describe what you expected.

I think the check should be that TotalAcquiredPairs is <= the number of rows in the aslcontext file, since at most the aslcontext file could have 2x the number of pairs, if all of the volumes in the file as control or label volumes, but at minimum there could just be a single pair with a bunch of m0scan, deltam, and/or cbf volumes filling out the rest of the asl file.

### BIDS specification section

https://github.com/bids-standard/bids-specification/blob/c2685ccc693fee58d3b03a24f5bd64bf3d1e4ff1/src/schema/rules/checks/asl.yaml#L199-L213

**Comment by effigies** [comment]
Yeah, the check should either be a bound or we could count pairs.
